ovos_plugin_manager/utils/__init__.py

```diff
@@ -32,6 +32,7 @@ class PluginTypes(str, Enum):
     AUDIO_TRANSFORMER = "neon.plugin.audio"
     QUESTION_SOLVER = "neon.plugin.solver"
     COREFERENCE_SOLVER = "intentbox.coreference"
+    KEYWORD_EXTRACTION = "intentbox.keywords"
```

Comment:
is 'intentbox' for backwards-compat or should this be neon/ovos namespaced?

Reply:
intentbox defines the namespace for a few plugins related to nlp, intentbox itelf is very WIP but i intend to use these in it

i guess i could change the namespace, just seemed appropriate since the other nlp tasks are under intentbox